datahub-web-react/src/app/analyticsDashboard/components/TimeSeriesChart.tsx

```diff
@@ -25,7 +25,7 @@ type Props = {
 const MARGIN_SIZE = 40;
 
 function insertBlankAt(ts: number, newLine: Array<NumericDataPoint>) {
-    const dateString = new Date(ts).toString();
+    const dateString = new Date(ts).toISOString();
```

Comment:
why this change here?

Reply:
For the tests basically. Otherwise it would contain people's timezone in the string and it become impossible to test this out for everyone.